src/defs.h

```diff
@@ -691,6 +691,7 @@ typedef enum { // must be coordinated with s_fds_handle_desc in common.cpp
 #ifndef __windows__
     POLL,
     EPOLL,
+    KQUEUE,
```

Comment:
I guess that `kqueue` should be under `defined(__FreeBSD__) || defined(__APPLE__)`